src/dataset.py

```diff
@@ -1,93 +1,94 @@
-from collections import defaultdict
-from csv import DictReader
 from decimal import Decimal
-from pathlib import Path
-from typing import Dict, Tuple
+from typing import Dict, Tuple, List, NamedTuple, Generator
 
 from flask import render_template
 from sodapy import Socrata
+import sqlite3
 
 
 # Set up the sodapy client
 client = Socrata("data.seattle.gov", None)
 LICENSE_DATASET = "enxu-fgzb"
 SALARY_DATASET = "2khk-5ukd"
-BADGE_DATASET_PATH = Path(__file__).parent / "data" / "SPD_roster_11-15-20.csv"
-_DataDict = Dict[str, Dict[str, str]]
-_DataNameLookup = Dict[str, _DataDict]
 
+# Connect to SQL database
+sql_conn = sqlite3.connect('./data/1-312-data.db')
+sql_curs = sql_conn.cursor()
+
+#################################################################################
+# Database format
+#################################################################################
+# Example:
+# Table
+# - column (datatype)
+#################################################################################
+# officers
+# - Serial (string)
+# - First (string)
+# - Middle (string)
+# - Last (string)
+# - TitleDescription (string)
+# - UnitDescription (string)
+#################################################################################
```

Comment:
This is good info, but I think we should have an actual `.sql` file with a `CREATE TABLE` statement for the sqlite table that we're using presently.